### ISSUE TITLE
Discovery on future home for accessibility documentation

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#44845]
## User Story
As a VFS team member, I want a central resource for finding information on accessibility best practices, sample code, and other accessibility tips.

As an accessibility specialist working on VA.gov, I want a central place to share best practices with VFS teams that I can help edit and maintain.

Assignee:
Peer Reviewer:

## Description
The [VA.gov experience standards](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/va-gov-experience-standards) map back to WCAG 2.1 guidelines. These live on the Platform website and are maintained by the Platform Governance team, and are the core of the Collaboration Cycle reviews for VFS products.

But there's more to accessibility than just WCAG. Best practices, code samples, links to external resources, and practical advice informed by accessibility specialists' experiences can be incredibly valuable to VFS teams working to build accessible products. This collected knowledge functions as an "accessibility cookbook" --- proven recipes for accessible patterns that can be used as a starting point for VFS teams.

Right now, there's no central resource for collecting this knowledge. Ideally, this should be a resource that is:
- Editable and maintainable by all accessibility specialists working in the VA.gov space, regardless of contract.
- Public to the web and visible to VA stakeholders and VFS teams.

Some possible candidates all have potential flaws:
- [Platform website](https://depo-platform-documentation.scrollhelp.site/) (powered by Confluence): Most of the accessibility specialists are not on Platform teams, and there are Confluence licensing limitations
- [Accessibility Community of Practice](https://vfs.atlassian.net/wiki/spaces/ACP/overview): Lives in Confluence, same user license issues
- Google Drive/Sites/Other Google product: Blocked for VA stakeholders.
- GitHub: Possibly cumbersome to edit; there's been a concerted effort to move away from GitHub as a documentation space.

When @briandeconinck and @shiragoodman discussed this they agreed none of these was a great fit, which leads us here. See also a [Slack discussion in #accessibility-ops](https://dsva.slack.com/archives/C023ZKDJJK1/p1657551831729109) on this topic as well.

This ticket is for discovery work to:
- Identify a candidate home for supplemental documentation that can be edited and maintained by all VA.gov accessibility specialists, OR
- Identify a candidate home for supplemental documentation that can only be edited by a subset of accessibility specialists, PLUS a proposed workflow for contributing 

## Impacted Artifacts
In addition to providing a space for future documentation, some existing items may be impacted:
- [Platform website information about accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-testing)
- [Draft a11y strategy document](https://docs.google.com/document/d/1YJHAmuTWDLPpvYG-E1IKMmKyCv76o8PwHuU7hcPi_UA/edit#heading=h.rgr9cd4bfeeo)
- [Draft a11y dashboard spreadsheet](https://docs.google.com/spreadsheets/d/1lgzMiZZ-0rBPDuB2gRupBAjsyF0WTe005VvEHYrMxww/edit#gid=228230969)
- [Best practices related to the deprecated wizard pattern](https://docs.google.com/document/d/12MctcX8uGg5iKEl8BedXWpsz1wSQY7ZPxW741RhHqsQ/edit?usp=sharing)
- [Old VSA Accessibility docs](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/teams/vsa/accessibility)
- Discussions that only happen in Slack but could/should be cleaned up into findable resources

In addition, this centralized accessibility resource should probably be linked to from Platform website. So some page(s) would eventually be touched there as well!

## Tasks
- [ ] _Placeholder task_

## Peer Review
- [ ] Ticket reassigned to peer reviewer when ready for peer review
- [ ] Peer Reviewer: Completes review and validates user story/Acceptance Criteria are met

## Acceptance Criteria
- [ ] A candidate home for supplemental accessibility documentation has been identified
- [ ] A possible workflow for editing and maintaining the documentation has been identified
- [ ] The proposal has been presented at the VA A11y Specialist Weekly Sync
- [ ] A follow-up issue has been created to begin building this resource

## How to prepare this issue
- [ ] Attach story to appropriate Initiative or Quarterly Tech-Debt Epic.
- [ ] Add User Story, description, tasks, and Acceptance Criteria
- [ ] Add any [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization).
- [ ] Once all above items are checked, add the 'Ready' label.
- [ ] Ticket has been discussed as a team
- [ ] Ticket has been sized as a team.
- [ ] Ticket has been assigned to a Sprint.
- [ ] Once assigned to the upcoming or current Sprint, move to the Planned pipeline.
- [ ] Once assigned to the upcoming or current Sprint, add assignee and specify Peer Reviewer.

**Comment by rsmithadhoc** [comment]
@briandeconinck Just a thought based on something we did recently for the Product Directory. This would definitely be more work but it allows the documentation to have one source of truth but publish to multiple locations.

- The [source of truth](https://github.com/department-of-veterans-affairs/product-directory/blob/main/product-directory.json) is in GitHub. This is JSON but I think individual Markdown files could work just as well.
  - To ease the pain of non-devs having to edit, we have an [issue template](https://github.com/department-of-veterans-affairs/product-directory/issues/new?assignees=cbonade&labels=&template=update-product-request.yml&title=Product+Directory+Change+Request+for+%5BProduct+Name%5D) and we generate a PR for them based on that.
- We [create Confluence pages](https://github.com/department-of-veterans-affairs/product-directory/blob/main/src/publish/publish.js) from that. This runs automatically via GitHub Actions for every merge.
- We are planning to move the [Confluence pages](https://vfs.atlassian.net/wiki/spaces/PWD/pages/2294120662/VFS+Product+Directory) into the appropriate space, so it will also publish to the Platform Website.

**Comment by humancompanion-usds** [comment]
@briandeconinck - Also for your consideration: An Accessibility section of the Design System. Much like we have a Content section we could have an Accessibility section. It would satisfy "Public to the web and visible to VA stakeholders and VFS teams." In addition, it would be a resource that other federal government agencies could reference. 

(Goes off and reads the Slack thread.) Okay, I see this was considered. For the sake of transparency, I asked DK to write up his doc on the deprecated Wizard pattern and plan to add that to that pattern (though could be convinced to put it elsewhere and link to it from there) in the Design System (and I will go file an issue to remind me of that now). In addition, I asked Sarah to document what we check for when doing an accessibility review, which she already did (amazing), on a component and plan to add that to the System as well. 

The USWDS has [this page](https://designsystem.digital.gov/design-principles/#embrace-accessibility) which lays out their high-level commitments. I hope to have at least something like that to bolster our [design principles](https://design.va.gov/about/principles#usable-by-everyone). But I'd like to do more. In short, I'd welcome more accessibility details in our Design System about how we do what we do for the audiences mentioned above to see and learn from.

**Comment by shiragoodman** [comment]
@jknipes @EWashb this ticket was discussed at Governance team refinement in January 2024. We still believe this is an issue, but we don't think it's within scope for Governance team. I have removed our label and added the Platform content label.

**Comment by EWashb** [comment]
I agree that we should have it represented on the Platform Website and that would fall to the content team.

**Comment by jknipes** [comment]
where ADE docs currently live:

![Image](https://github.com/user-attachments/assets/52d1e016-9d79-4ca9-b999-3e9007f729f5)
